newrelic/events_service.go

```diff
@@ -0,0 +1,14 @@
+package newrelic
+
+import (
+	"context"
+)
+
+type EventsAPI interface {
```

Comment:
For our future selves, can you add a doc block above this that explains what this is and why it's here please? 🙂

Reply:
done